### PR TITLE
Add logic for login method 

### PR DESCRIPTION
**Pull request by fer-pucheta-tarmac**

### CODE REVIEW

File: cypress/support/helpers/e2e/commonHelpersE2E.js [hunk 1]
```diff
@@ -1,3 +1,25 @@
 export const isE2ESuite = () => {
     return Cypress.spec.relative.includes('-e2e')
+}
+
+export const getUserCredentials = (user) => {
+    let userEmail = Cypress.env('USER_DEV_EMAIL')
```
Comment (alexis-moreno-0) [inline cypress/support/helpers/e2e/commonHelpersE2E.js:6]: you are doing this again below

File: cypress/support/stepDefinitions/e2e/sectionValidationE2E.js [hunk 2]
```diff
@@ -1,15 +1,43 @@
...
+
+    loginSelectors.getSignInBtn().click()
+
+    const sentArgs = { username: params.userEmail, password: params.pwd }
+
+    cy.origin('https://accounts.google.com', { args: sentArgs }, ({ username, password }) => {
+        cy.on('uncaught:exception', () => false)
+        cy.url().should('contain', 'accounts.google.com')
+
+        cy.get('input[type="email"]').type(username)
```
Comment (alexis-moreno-0) [inline cypress/support/stepDefinitions/e2e/sectionValidationE2E.js:27]: perhaps I would move selectors to a selector class

File: cypress/support/stepDefinitions/e2e/sectionValidationE2E.js [hunk 3]
```diff
@@ -1,15 +1,43 @@
...
+        cy.get('#passwordNext button').click()
+    })
+
+    cy.url().should('contain', globalUrl)
 })
 
 Then('I should see following sections', (dataTable) => {
     const sections = Array.from(dataTable.hashes(), row => row.section)
 
+    cy.get(basicSelectors.getSectionsSelector(), { timeout: 6000 }).should('be.visible')
```
Comment (alexis-moreno-0) [inline cypress/support/stepDefinitions/e2e/sectionValidationE2E.js:40]: I would put the cy.get on the selector class, is the timeout needed?

File: cypress/support/stepDefinitions/e2e/signInTestE2E.js [hunk 4]
```diff
@@ -1,10 +1,10 @@
 import { Given, Then } from '@badeball/cypress-cucumber-preprocessor'
-import signInPage from '../../pageObjects/signInPage'
+import loginSelectors from '../../pageObjects/loginSelectors'
 
 Given('I navigate to Tarmac internal tool', () => {
```
Comment (alexis-moreno-0) [inline cypress/support/stepDefinitions/e2e/signInTestE2E.js:4]: "Tarmac Internal Tool"

File: cypress/support/stepDefinitions/e2e/signInTestE2E.js [hunk 5]
```diff
@@ -1,10 +1,10 @@
...
-    signInPage.enterURL(globalUrl)
+    loginSelectors.enterURL(globalUrl)
 })
 
 Then('I should see the google sign in button', () => {
```
Comment (alexis-moreno-0) [inline cypress/support/stepDefinitions/e2e/signInTestE2E.js:8]: Google

File: cypress/support/stepDefinitions/e2e/sectionValidationE2E.js (hunk 3 above)
Comment (fer-pucheta-tarmac) [reply cypress/support/stepDefinitions/e2e/sectionValidationE2E.js:40]: if I move to a selector class I can't put the timeout for waiting for the selector. I put it in this way because takes time between login and seeing this selector

File: cypress.config.js [hunk 6]
```diff
@@ -17,6 +19,6 @@ module.exports = defineConfig({
 
 			return config
 		},
-		specPattern: "cypress/e2e/**/*.feature",
+		specPattern: ['cypress//**/*.{feature,features}'],
```
Comment (bramondelli-tarmac) [inline cypress.config.js:22]: why did you add the "features" as specPattern?

File: cypress/e2e/smokeE2E/signInTestE2E.feature [hunk 7]
```diff
@@ -3,5 +3,5 @@ Feature: Sign In Test
   Testing Sign In page
 
   Scenario: I want to verify content in Sign In Page
-    Given I navigate to Tarmac internal tool
-    Then I should see the google sign in button
\ No newline at end of file
+    Given I navigate to Tarmac Internal Iool
```
Comment (bramondelli-tarmac) [inline cypress/e2e/smokeE2E/signInTestE2E.feature:6]: Internal Iool --> Internal Tool

File: cypress/support/stepDefinitions/e2e/sectionValidationE2E.js (hunk 2 above)
Comment (bramondelli-tarmac) [reply cypress/support/stepDefinitions/e2e/sectionValidationE2E.js:27]: could this code be moved to the commands file??

**Review by bramondelli-tarmac — APPROVED** [review]
